### PR TITLE
Feature/styles

### PR DESCRIPTION
**Pull request by oacortez**
What (if any) features are you implementing?
- Completed styling for our mobile view 
- Add button to end round early
- Created a end round early component

What (if anything) did you refactor?
- n/a

Were there any issues that arose?
- button on the bottom for end round overview needs to be adjusted

Is there anything that you need from your teammate?
- please review!

Any other comments, questions, or concerns? 
- n/a

### CODE REVIEW

File: src/Components/EndCurrentRoundModal.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import '../Styles/EndCurrentRoundModal.scss'
+
+
+const EndCurrentRoundModal = ({toggleConfirm, endCurrentRound}) => {
```
Comment (philalewis) [inline src/Components/EndCurrentRoundModal.js:5]: This adds some more functionality. The option to end early is a good feature.

File: src/Components/Searchbar.js [hunk 2]
```diff
@@ -17,13 +17,18 @@ const Searchbar = () => {
     navigate(newURL, {state: newURL})
   }
 
+  const handleKeyUp = (event) => {
```
Comment (philalewis) [inline src/Components/Searchbar.js:20]: Good to be able to just hit enter when searching

File: src/Styles/EndCurrentRoundModal.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgba(0,0,0,0.4);
+  display: flex;
+  justify-content: space-between;
+}
+
+.confirm-deletion-modal {
```
Comment (philalewis) [inline src/Styles/EndCurrentRoundModal.scss:17]: This modal looks great.

File: src/Styles/Player.scss [hunk 4]
```diff
@@ -4,10 +4,23 @@
...
 .plus-btn {
   @include navy-button();
+  width: 30px;
+  height: 30px;
 }
 
 .player-score-input {
+  display: grid;
+  text-align: center;
+  grid-template-columns: 3fr repeat(3, 1fr);
```
Comment (philalewis) [inline src/Styles/Player.scss:14]: This makes the first box take up half of the available space. Looks good.

**Review by philalewis — APPROVED** [review]

File: src/Components/Scorecard.js [hunk 5]
```diff
@@ -113,41 +119,38 @@ const Scorecard = () => {
...
     endRound(courseInfo.roundId)
-    .then(data => navigate('/round-overview', {state: data}))
+    .then(data => {
+      setScorecard([])
+      navigate('/round-overview', {state: data})
+    })
     .catch(error => setErrorMessage(error))
   }
 
-  // const displayItems = scorecard.length > 0 ?
```
Comment (GraceGardner) [inline src/Components/Scorecard.js:120]: good catch

File: src/Styles/EndCurrentRoundModal.scss [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  display: flex;
```
Comment (GraceGardner) [inline src/Styles/EndCurrentRoundModal.scss:18]: May be an opportunity for a mixin. The modals all have a lot of the same styling

**Review by GraceGardner — APPROVED** [review]
Nice work

**Pull request merged**